src/ceph-detect-init/tests/test_all.py

```diff
@@ -56,6 +60,10 @@ def test_debian(self):
                                  distro='ubuntu',
                                  codename='vivid'):
             self.assertEqual('systemd', debian.choose_init())
+        with mock.patch.multiple('ceph_detect_init.debian',
+                                 distro='not-debian',
```

Comment:
"not-debian" will actually match because

     if distro.lower() in ('debian'):

should be

    if distro.lower() in ('debian',):

or, less confusing:

    if distro.lower() == 'debian':